doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,599 @@
+    * Retransmissions are now supported when transmission security is handled by the radio driver.
+    * Support for CSL Accuracy TLV in the MLE Parent Response.
+    * Link Metrics data is now properly updated when using ACK-based Probing.
+
+  * Added support for Thread Backbone Border Router on the :ref:`thread_architectures_designs_cp_rcp` architecture.
+
+* Updated:
+
+  * :ref:`ot_cli_sample` sample - Updated with the USB support.
+  * :ref:`ot_coprocessor_sample` sample - Updated with the USB support.
+  * Removed ``NET_SHELL`` from the Thread samples due to its limited usefulness.
+
+Zigbee
+------
+
+* In this release, Zigbee is supported for development and should not be used for production.
+  The |NCS| v1.5.1 contains the certified Zigbee solution supported for production.
+* Added:
+
+  * Added production support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF52 Series devices and nRF21540 EK.
+  * Added development support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF53 Series devices and nRF21540 EK.
+  * Added development support for ``nrf5340dk_nrf5340_cpuapp`` to the :ref:`zigbee_ncp_sample` sample.
+  * :ref:`lib_zigbee_zcl_scenes` library with documentation.
+    This library was separated from the Zigbee light bulb sample.
+  * :ref:`zigbee_template_sample` sample.
+    This minimal Zigbee router application can be used as the starting point for developing custom Zigbee devices.
+  * Added user guide about :ref:`ug_zigee_adding_clusters` that is based on the new template sample.
+  * Added API for vendor-specific NCP commands.
+    See the :ref:`Zigbee NCP sample <zigbee_ncp_vendor_specific_commands>` page for more information.
+  * Added API for Zigbee command for getting active nodes.
+
+* Updated:
+
+  * ZBOSS Zigbee stack to version 3.8.0.1+4.0.0.
+    See the :ref:`nrfxlib:zboss_changelog` in the nrfxlib documentation for detailed information.
+  * :ref:`ug_zigbee_tools_ncp_host` is now supported for production and updated to version 1.0.0.
+  * :ref:`zigbee_ug_logging_stack_logs` - Improved printing ZBOSS stack logs.
+    Added new backend options to print ZBOSS stack logs with option for using binary format.
+  * Fixed the KRKNWK-9743 known issue where the timer could not be stopped in Zigbee routers and coordinators.
+  * Fixed the KRKNWK-10490 known issue that would cause a deadlock in the NCP frame fragmentation logic.
+  * Fixed the KRKNWK-6071 known issue with inaccurate ZBOSS alarms.
+  * Fixed the KRKNWK-5535 known issue where the device would assert if flooded with multiple Network Address requests.
+  * Fixed an issue where the NCS would assert in the host application when the host started just after SoC's SysReset.
+
+ESB
+---
+
+* Updated:
+
+  * Modified the ESB interrupts configuration to reduce the ISR latency and enable scheduling decision in the interrupt context.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added:
+
```

Comment:
```suggestion
  * Updated ``NRF_802154_SWI_PRIORITY``, ``NRF_802154_ECB_PRIORITY`` and ``NRF_802154_SL_RTC_IRQ_PRIORITY`` to comply with the limitation introduced in :ref:`nrfxlib:nrf_802154_driver`.
```

Not sure if the syntax is correct, but this entry is related with https://github.com/nrfconnect/sdk-nrf/pull/5640.